gulpfile.js

```diff
@@ -52,6 +62,24 @@ gulp.task('copy:html', function () {
     .pipe(gulp.dest(paths.html.dest));
 });
 
+// ** Distribution ** //
+gulp.task('dist', ['dist:scripts']);
+
+// Process scripts and concatenate files into one output file
+gulp.task('dist:scripts', ['dist:clean'], function() {
+ return gulp.src(paths.dist.srcJS)
+ .pipe(jshint())
+ .pipe(jshint.reporter('default'))
+ .pipe(uglify())
+ //.pipe(concat('ea-timeline.js'))
```

Comment:
I will pull it in now. But this should be addressing #20 and creating a proper dist folder structure (CSS, source maps,etc.).